core/src/crypto/AESCipher.cpp

```diff
@@ -57,7 +57,6 @@ namespace ledger {
 
                 // Create an IVt
                 auto IV = _rng->getRandomBytes(AES256::BLOCK_SIZE);
-                assert(IV.size() == AES256::BLOCK_SIZE);
```

Comment:
Nope. You can’t do that. 🙅‍♂️ That assert is needed.

Reply:
ok, put it back

Reply:
🔞